<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCCurve</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DCCurve</string>
   </property>
  </widget>
  <widget class="QLabel" name="usuario_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>221</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>USUARIO   1</string>
   </property>
  </widget>
  <widget class="QLabel" name="puntaje_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="usuario_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>251</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>USUARIO   2</string>
   </property>
  </widget>
  <widget class="QLabel" name="puntaje_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="usuario_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>281</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>USUARIO   3</string>
   </property>
  </widget>
  <widget class="QLabel" name="puntaje_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="usuario_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>311</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>USUARIO   4</string>
   </property>
  </widget>
  <widget class="QLabel" name="puntaje_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>19</y>
     <width>661</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_top"/>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>19</y>
     <width>661</width>
     <height>501</height>
    </rect>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>501</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 4px;
border: 8px solid black;
border-color: rgb(0, 255, 255);
</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="juego_layout"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="fondo.qrc"/>
 </resources>
 <connections/>
</ui>
